index.d.ts

```diff
@@ -0,0 +1,123 @@
+declare module "fetch-ponyfill" {
+    export default FetchPonyfill.bootstrap;
```

Comment:
All the documentation I find says that this should be `export = FetchPonyfill.bootstrap` but yet that does not work for me, only this does.